src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -866,3 +866,51 @@ public sealed partial class OperationAbortedException : System.SystemException
         internal OperationAbortedException() { }
     }
 }
+namespace System.Data.SqlTypes
+{
+    public sealed partial class SqlFileStream : System.IO.Stream
+    {
+        public SqlFileStream
+            (
+                string path,
+                byte[] transactionContext,
+                System.IO.FileAccess access
+            )
+        { }
+
+        public SqlFileStream
```

Comment:
Nit: This declaration should be in a single line, following the convention of the file.